includes/forms.inc

```diff
@@ -515,3 +537,20 @@ function islandora_ip_embargo_object_embargo_form_submit($form, &$form_state) {
   }
   return $form;
 }
+
+/**
+ * Validate hexidecimal color input.
+ *
+ * @param array $element
+ *   The Drupal element data.
+ * @param array $form_state
+ *   The Drupal form state.
+ * @param array $form
+ *   The Drupal form definition.
+ */
+function islandora_ip_embargo_hex_validate($element, &$form_state, $form) {
+  $color = $element['#value'];
+  if (!color_valid_hexadecimal_string($color)) {
```

Comment:
This adds an undocumented/unenforced dependency on the color module. Since the referenced function is a straight-forward one-liner, I think a copy-pasta is fine rather than introducing the dependency.